packages/devextreme/testing/testcafe/tests/editors/textArea/index.ts

```diff
@@ -1,101 +1,101 @@
     value: text,
-  }));
-
-  test(`TextArea has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and maxHeight option is 5em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('5em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql(autoResizeEnabled ? 'auto' : '')
-      .expect(textArea.element.getStyleProperty('height'))
-      .eql(autoResizeEnabled ? '70px' : '55px')
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql(autoResizeEnabled ? '68px' : '');
-  }).before(async () => createWidget('dxTextArea', {
-    maxHeight: '5em',
-    autoResizeEnabled,
-    width: 200,
+  };
+
+  await appendElementTo('#container', 'div', 'textArea1');
+  await appendElementTo('#container', 'div', 'textArea2');
+
+  await createWidget('dxTextArea', {
+    ...config,
+    autoResizeEnabled: true,
+  }, '#textArea1');
+
+  await createWidget('dxTextArea', {
+    ...config,
+    autoResizeEnabled: false,
+  }, '#textArea2');
+});
+
+test('TextArea has correct height autoResizeEnabled and different value length', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await testScreenshot(t, takeScreenshot, 'TextArea has correct height autoResizeEnabled and different value length.png', { element: '#container' });
+
+  await t
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+}).before(async () => {
+  await setAttribute('#container', 'style', 'width: 300px;');
+
+  const config = {
+    width: '100%',
     value: text,
-  }));
-
-  test(`TextArea with font-size style has correct height with "autoResizeEnabled" is ${autoResizeEnabled} and maxHeight option is 5em`, async (t) => {
-    const textArea = new TextArea('#container');
-
-    await t
-      .expect(getPropertyValue(textArea.element, 'maxHeight'))
-      .eql('5em')
-      .expect(getPropertyValue(textArea.element, 'height'))
-      .eql(autoResizeEnabled ? 'auto' : '')
-      .expect(parseInt(await textArea.element.getStyleProperty('height'), 10))
-      .eql(autoResizeEnabled ? 60 : 49)
-      .expect(getPropertyValue(textArea.input, 'height'))
-      .eql(autoResizeEnabled ? '58px' : '');
-  }).before(async () => {
-    await insertStylesheetRulesToPage('#container { font-size: 12px; }');
-
-    return createWidget('dxTextArea', {
-      maxHeight: '5em',
-      autoResizeEnabled,
-      width: 200,
-      value: text,
-    });
-  });
+    autoResizeEnabled: true,
+  };
+
+  await appendElementTo('#container', 'div', 'textArea1');
+  await appendElementTo('#container', 'div', 'textArea2');
+
+  await createWidget('dxTextArea', {
+    ...config,
+  }, '#textArea1');
+
+  await createWidget('dxTextArea', {
+    ...config,
+    value: text + text,
+  }, '#textArea2');
+});
+
+test('Bug Fix (T1221869) when there is maxHeight and autoResizeEnabled', async (t) => {
```

Comment:
Usually the bug ID we placed in the end of test name. It's better if we call this test as following:

`Height of TextArea input should have the correct height when the maxHeight option is set to 80px`.

The screenshot name:
`TextArea appearance when maxHeight='80px'.png'